terminus-ui/spacing/src/vertical-spacing.directive.spec.ts

```diff
@@ -57,4 +93,15 @@ describe(`TsVerticalSpacingDirective`, function() {
 
   });
 
+  describe(`default TsVerticalSpacing`, () => {
+    test(`should set default values`, () => {
+      const fixtureBasic = createComponent(TestHostBasicComponent, [], [TsSpacingModule]);
+      fixtureBasic.detectChanges();
+
+      expect(fixtureBasic).toBeTruthy();
+      const spacingDiv = fixtureBasic.debugElement.query(By.directive(TsVerticalSpacingDirective));
+      expect(spacingDiv).toBeTruthy();
```

Comment:
this tests `spacingDiv` existence. but it doesn't test the default value?

Reply:
Thank you, Wendy. It does not have a class, but I did find another way to check!